npm/extensions.bzl

```diff
@@ -231,7 +231,6 @@ WARNING: Cannot determine home directory in order to load home `.npmrc` file in
             commit = i.commit,
             custom_postinstall = i.custom_postinstall,
             deps = i.deps,
-            dev = i.dev_only,
```

Comment:
This is also key. The `npm_import(dev)` and `NpmPackageStoreInfo.dev` flags are gone along with the `npm_imported_package_store_internal(dev_only)` and the `struct(dev_only)`.